src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.test.tsx

```diff
@@ -62,6 +72,11 @@ describe('<TranscriptsListItemInfo /', () => {
 
   beforeEach(() => {
     wrapper = renderComponent();
+
+    (useQuery as any).mockImplementation(() => ({
+      data: null,
+      loading: true
+    }));
```

Comment:
Would it work if the whole `.mockImplementation` thing for useQuery is moved up into the `jest.mock('@apollo/client'` factory function? `useQuery: jest.fn().mockImplementation ...`?

Reply:
✅